numpyro/infer/util.py

```diff
@@ -269,7 +269,7 @@ def init_to_value(values):
 
 
 def find_valid_initial_params(rng, model, *model_args, init_strategy=init_to_uniform(),
-                              param_as_improper=False, prototype_params=None, **model_kwargs):
+                              param_as_improper=False, prototype=None, **model_kwargs):
```

Comment:
How about also taking `inv_transforms` (maybe calling it `support_transforms` or `constraint_transforms`) as input instead of having a tuple for `prototype`?